pkg/server/server.go

```diff
@@ -87,6 +88,7 @@ type Server struct {
 	GitProviderService       gitproviders.IGitProviderService
 	ProviderManager          manager.IProviderManager
 	ProfileDataService       profiledata.IProfileDataService
+	Provider                 provider.Provider
```

Comment:
Remove this

Reply:
if i remove this, how do i access the `checkRequirement()`

Reply:
Providers are registered in the `registerProviders` function. https://github.com/daytonaio/daytona/pull/1300#discussion_r1827630929

Reply:
Okay, i just pushed changes now